pandas/core/apply.py

```diff
@@ -1427,14 +1427,17 @@ def _managle_lambda_list(aggfuncs: Sequence[Any]) -> Sequence[Any]:
     if len(aggfuncs) <= 1:
         # don't mangle for .agg([lambda x: .])
         return aggfuncs
-    i = 0
+
     mangled_aggfuncs = []
-    for aggfunc in aggfuncs:
-        if com.get_callable_name(aggfunc) == "<lambda>":
-            aggfunc = partial(aggfunc)
-            aggfunc.__name__ = f"<lambda_{i}>"
-            i += 1
-        mangled_aggfuncs.append(aggfunc)
+    for idx, aggfunc in enumerate(aggfuncs):
+        if not com.get_callable_name(aggfunc) == "<lambda>":
+            mangled_aggfuncs.append(aggfunc)
+            continue
+
+        partial_aggfunc = aggfunc
+        partial_aggfunc = partial(aggfunc)
+        partial_aggfunc.__name__ = f"<lambda_{idx}>"
```

Comment:
also, I don't think the logic is right here, please try to keep the logic the same as the original

Reply:
The original one was:
```
i = 0
for aggfunc in aggfuncs:
        if com.get_callable_name(aggfunc) == "<lambda>":
            aggfunc = partial(aggfunc)
            aggfunc.__name__ = f"<lambda_{i}>"
            i += 1
        mangled_aggfuncs.append(aggfunc)
```

I removed the variable `i`, replacing it by `idx` generated by the call to `enumerate`. Then I inverted the `if` clause, making it a `if not`. In this way I can insert inside the condition the single line:
`mangled_aggfuncs.append(aggfunc)`
and then continue the loop. So I moved what was inside the if outside. Sorry I understand it's not super clear, but I thought would look nicer to have a lighter content inside the if statement. If you want I can revert the logic.